src/v/kafka/server/server.cc

```diff
@@ -720,6 +721,16 @@ ss::future<response_ptr> sasl_handshake_handler::handle(
         }
     }
 
+    if (supports("PLAIN")) {
+        supported_sasl_mechanisms.emplace_back(
+          security::plain_authenticator::name);
+        if (request.data.mechanism == security::plain_authenticator::name) {
+            ctx.sasl()->set_mechanism(
+              std::make_unique<security::plain_authenticator>(
+                ctx.credentials()));
+        }
+    }
```

Comment:
discussion: Getting clarification from REQ-01.  Not sure if the wording was wrong or if `PLAIN` implies also enabling `SCRAM`.

Reply:
PLAIN means enable PLAIN. One other secure mechanism must be explicitly enabled.